_layouts/default.html

```diff
@@ -5,7 +5,7 @@
     <div class="row">
       <div class="col-xs-12 col-sm-8">
         <ul class="post-list">
-          {% for post in site.posts reversed %}
```

Comment:
@Lykathia turns out reverse chronological is default, so I reversed the reverse and hadn't intended to. Was difficult to see before adding this third post since the first two were authored on the same date.